diff -uwr a/config/routes.rb b/config/routes.rb
--- a/config/routes.rb	2017-04-06 15:38:29.241504647 +0200
+++ b/config/routes.rb	2017-04-06 15:41:40.515427162 +0200
@@ -16,7 +16,7 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
 Rails.application.routes.draw do
-  root :to => 'welcome#index', :as => 'home'
+  root :to => 'projects#index', :as => 'home'
 
   match 'login', :to => 'account#login', :as => 'signin', :via => [:get, :post]
   match 'logout', :to => 'account#logout', :as => 'signout', :via => [:get, :post]
