diff -uwr a/app/helpers/projects_helper.rb b/app/helpers/projects_helper.rb
--- a/app/helpers/projects_helper.rb	2015-09-20 08:20:43.000000000 +0200
+++ b/app/helpers/projects_helper.rb	2017-03-24 23:04:03.265306002 +0100
@@ -64,7 +64,7 @@
   # Renders the projects index
   def render_project_hierarchy(projects)
     render_project_nested_lists(projects) do |project|
-      s = link_to_project(project, {}, :class => "#{project.css_classes} #{User.current.member_of?(project) ? 'my-project' : nil}")
+      s = link_to_project(project, { :jump => "wiki" }, :class => "#{project.css_classes} #{User.current.member_of?(project) ? 'my-project' : nil}")
       if project.description.present?
         s << content_tag('div', textilizable(project.short_description, :project => project), :class => 'wiki description')
       end
