diff -uwr a/plugins/scrum/lib/scrum/issue_query_patch.rb b/plugins/scrum/lib/scrum/issue_query_patch.rb
--- a/plugins/scrum/lib/scrum/issue_query_patch.rb	2017-07-22 19:58:35.714793456 +0000
+++ b/plugins/scrum/lib/scrum/issue_query_patch.rb	2017-07-22 19:58:19.742792799 +0000
@@ -36,14 +36,14 @@
         alias_method_chain :initialize_available_filters, :scrum
 
         def issues_with_scrum(options = {})
-          options[:include] ||= {}
+          options[:include] ||= []
           options[:include] << :sprint
           issues_without_scrum(options)
         end
         alias_method_chain :issues, :scrum
 
         def issue_ids_with_scrum(options = {})
-          options[:include] ||= {}
+          options[:include] ||= []
           options[:include] << :sprint
           issue_ids_without_scrum(options)
         end
diff -uwr a/plugins/scrum/lib/scrum/projects_helper_patch.rb b/plugins/scrum/lib/scrum/projects_helper_patch.rb
--- a/plugins/scrum/lib/scrum/projects_helper_patch.rb	2017-07-22 19:59:23.202795410 +0000
+++ b/plugins/scrum/lib/scrum/projects_helper_patch.rb	2017-07-22 19:59:11.198794916 +0000
@@ -17,7 +17,10 @@
           if User.current.allowed_to?(:manage_sprints, @project)
             index = tabs.index({:name => 'versions', :action => :manage_versions,
                                 :partial => 'projects/settings/versions',
-                                :label => :label_version_plural})
+                                :label => :label_version_plural,
+                                :url => {:tab => 'versions',
+                                         :version_status => params[:version_status],
+                                         :version_name => params[:version_name]}})
             if index
               tabs.insert(index,
                           {:name => 'product_backlogs', :action => :edit_sprints,
